src/PKSim.CLI.Core/Services/JsonSimulationRunner.cs

```diff
@@ -132,7 +132,7 @@ private async Task runAndExportSimulationsInSnapshotFile(FileInfo projectFile, s
          _logger.AddInfo($"Starting batch simulation export for file '{projectFile}'");
          try
          {
-            var project = await _snapshotTask.LoadProjectFromSnapshotFileAsync(projectFile.FullName);
+            var project = await _snapshotTask.LoadProjectFromSnapshotFileAsync(projectFile.FullName, runSimulations:false);
```

Comment:
This means that for example **PKSim.BatchRunner** was also executing all simulation runs twice?